lab_2_tokenize_by_bpe/main.py

```diff
@@ -69,6 +137,20 @@ def get_vocabulary(
     :param unknown_token: a token to signify an unknown token
     :return: dictionary in the form of <token: identifier>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(unknown_token, str):
+        return None
+    listed_tokens = set()
+    token_identifier_dict = {}
+    for word in word_frequencies:
+        for token in word:
+            listed_tokens.add(token)
+            for element in token:
+                listed_tokens.add(element)
+    listed_tokens.add(unknown_token)
+    sorted_tokens = sorted(listed_tokens, key=lambda x: (-len(x), x))
```

Comment:
Set are unordered structures, let's not sort it

Reply:
Fixed (If I understood the problem correctly)